components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -500,6 +500,7 @@ internal class DisplayToolbar(
     }
 
     companion object {
+        internal const val MEASURED_HEIGHT_THIRD_DENOMINATOR = 3
         internal const val MEASURED_HEIGHT_DENOMINATOR = 2
```

Comment:
Is `MEASURED_HEIGHT_DENOMINATOR` now unused?

Reply:
No, it's used to divide the `totalPadding` in half.